pkgs/tools/misc/osm2pgsql/default.nix

```diff
@@ -21,13 +21,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "osm2pgsql";
-  version = "1.9.0";
+  version = "1.9.2";
 
   src = fetchFromGitHub {
-    owner = "openstreetmap";
+    owner = "osm2pgsql-dev";
```

Comment:
Should osm2pgsql blog post [New Github Organisation](https://osm2pgsql.org/news/2023/09/20/new-github-organisation.html) maybe be mentioned in the commit message or at least in the PR description?

Reply:
Added to PR description